packages/iris-grid/src/IrisGridModel.ts

```diff
@@ -488,6 +488,20 @@ abstract class IrisGridModel<
    */
   abstract delete(ranges: GridRange[]): Promise<void>;
 
+  abstract seekRow(
+    startRow: number,
+    column: Column,
+    valueType: string,
```

Comment:
Should restrict this to `VariableTypeUnion` (see my other comment about defining a `VariableType` type)